### DISCUSSION TITLE
change manual mesh 3x3 to 5x5 in firmware

### DISCUSSION THREAD

**Discussion by mosheas** [Jyers/Marlin#1687] [Q&A]
How can I change manual mesh 3x3 to 5x5 in firmware?

**Comment by vhristenco** [comment]
GRID_MAX_POINTS_X and GRID_MAX_POINTS_Y in configuration.h

**Comment by Cal-LF** [comment]
Change 1606 #define GRID_MAX_POINTS_X 3 4 or 5 to get the grid size you want. Compile. Y dimension will follow the X.

#if EITHER(AUTO_BED_LEVELING_LINEAR, AUTO_BED_LEVELING_BILINEAR)

  // Set the number of grid points per dimension.
  #define GRID_MAX_POINTS_X 3
  #define GRID_MAX_POINTS_Y GRID_MAX_POINTS_X

**Reply by mosheas** [reply]
Yes thank you, I did it, everything works well. 🙏
